src/manifolds/PositiveNumbers.jl

```diff
@@ -53,6 +53,23 @@ function check_point(M::PositiveNumbers, p; kwargs...)
     return nothing
 end
 
+@doc raw"""
+    change_metric(M::PositiveNumbers, E::EuclideanMetric, p, X)
+
+Given a tangent vector ``X ∈ T_p\mathcal M``representing a gradient with respect to the [`EuclideanMetric`](@ref) `g_E`,
+this function changes into the positivity metric representation of
+[`PositiveNumbers`](lref) `M` for the same gradient.
```

Comment:
What's `lref`?

Reply:
alt+l on a German Mac keyboard is the @ - sorry.